server/py/services/api/db/sqldb/db.py

```diff
@@ -5384,6 +5384,90 @@ def _enrich_notification(_notification):
             for cooldown, notification in zip(cooldowns, notifications)
         ]
 
+    @staticmethod
+    def create_partitions(
+        session: Session,
+        table_name: str,
+        partitioning_information_list: list[tuple[str, str, str]],
+    ):
+        """
+        Creates partitions in the specified database table.
+
+        :param session: SQLAlchemy session for database connection.
+        :param table_name: Name of the table where partitions will be created.
+        :param partitioning_information_list: List of tuples, each containing:
+            - partition_name: The name for the partition.
+            - partition_value: The "LESS THAN" boundary value for the partition.
+            - partition_expression: The SQL partition expression.
+        """
+        alter_table_template = f"""
+            ALTER TABLE {table_name}
+            PARTITION BY RANGE ({partitioning_information_list[0][2]}) (
```

Comment:
Why are you always using the same value here?

Reply:
@TomerShor because the structure of partitioning_information_list is always the same

Reply:
Actually, I decided to rework this logic a bit to avoid keeping the same value N times